### PR TITLE
ADD: execution clients - Besu, Nethermind

### PR DESCRIPTION
**Pull request by gbayasgalan**
resolves: https://github.com/stereum-dev/ethereum-node/issues/315, https://github.com/stereum-dev/ethereum-node/issues/312

### CODE REVIEW

File: controls/defaults/stereum_defaults.yaml [hunk 1]
```diff
@@ -15,6 +15,8 @@ stereum_static:
 
       # execution clients
       geth: v1.10.19
+      besu: latest
```
Comment (stefa2k) [inline controls/defaults/stereum_defaults.yaml:18]: we should use a static tag instead, latest stable version I found is `22.4.3`

File: controls/roles/manage-service/molecule/besu/prepare.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        include_role:
+          name: "manage-service"
+      vars:
+        stereum_args:
+          manage_service:
+            save: true
+            state: started
+            configuration:
+              id: "{{ beacon_service }}"
+              image: sigp/lighthouse:v2.1.2
```
Comment (stefa2k) [inline controls/roles/manage-service/molecule/besu/prepare.yml:45]: please use `stereum_static.defaults.versions`

File: controls/roles/manage-service/molecule/nethermind/prepare.yml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        include_role:
+          name: "manage-service"
+      vars:
+        stereum_args:
+          manage_service:
+            save: true
+            state: started
+            configuration:
+              id: "{{ beacon_service }}"
+              image: sigp/lighthouse:v2.1.2
```
Comment (stefa2k) [inline controls/roles/manage-service/molecule/nethermind/prepare.yml:45]: please use `stereum_static.defaults.versions`

**Review by stefa2k — CHANGES_REQUESTED** [review]
this doesn't resolve #312 #315 yet, Stereum Launcher still needs to integrate it after this

File: controls/roles/manage-service/molecule/besu/prepare.yml (hunk 2 above)
Comment (gbayasgalan) [reply controls/roles/manage-service/molecule/besu/prepare.yml]: done!

File: controls/roles/manage-service/molecule/nethermind/prepare.yml (hunk 3 above)
Comment (gbayasgalan) [reply controls/roles/manage-service/molecule/nethermind/prepare.yml]: done!

**Review by stefa2k — APPROVED** [review]

**Pull request merged**